### ISSUE TITLE
Crash when opening storage vessel

### ISSUE THREAD

**Issue by Leden** [anegostudios/VintageStory-Issues#324]
**Game Version:** 1.10.6
**Platform:** Windows
**Modded:** No 

### Description
Try opening a storage vessel in which a pot with rotten food resides. Game crashes.

### How to reproduce
1. Make a pot of food
2. Put it in a storage vessel and wait for it to rot
3. Open the storage vessel
4. It crashes.

### Expected behavior
Does not crash, but show the UI of the container.

### Logs
```
Running on 64 bit Windows with 8 GB RAM
Version: v1.10.6 (Stable)19/09/15 5:24:26 PM: Critical error occurred
System.NullReferenceException: Ссылка на объект не указывает на экземпляр
объекта.
в Vintagestory.GameContent.MealMeshCache.GenMealMesh(CookingRecipe
forRecipe, ItemStack[] contentStacks, Vec3f foodTranslate)
в Vintagestory.GameContent.MealMeshCache.GenMealInContainerMesh(Block
containerBlock, CookingRecipe forRecipe, ItemStack[] contentStacks, Vec3f
foodTranslate)
в
Vintagestory.GameContent.MealMeshCache.GetOrCreateMealInContainerMeshRef(Block
containerBlock, CookingRecipe forRecipe, ItemStack[] contentStacks, Vec3f
foodTranslate)
в Vintagestory.GameContent.BlockMeal.OnBeforeRender(ICoreClientAPI capi,
ItemStack itemstack, EnumItemRenderTarget target, ItemRenderInfo&
renderinfo)
в
Vintagestory.Client.NoObf.InventoryItemRenderer.GetItemStackRenderInfo(_ahKHCLkUrtGs3H3UuMzDUmqqXDB
game, ItemSlot inSlot, EnumItemRenderTarget target)
в
Vintagestory.Client.NoObf.InventoryItemRenderer.RenderItemstackToGui(ItemSlot
inSlot, Double posX, Double posY, Double posZ, Single size, Int32 color,
Boolean shading, Boolean rotate, Boolean showStackSize)
в
Vintagestory.API.Client.GuiElementItemSlotGridBase.RenderInteractiveElements(Single
deltaTime)
в Vintagestory.API.Client.GuiComposer.Render(Single deltaTime)
в Vintagestory.API.Client.GuiDialog.OnRenderGUI(Single deltaTime)
в Vintagestory.API.Client.GuiDialogBlockEntity.OnRenderGUI(Single
deltaTime)
в _uA2VTOGNqbAsrYr3suTo8NVVnbf._FzrZiSc0F3yZCAxfhPbeybFO8nW(Single )
в
_ee95dOUYyEAX8VgbT4J2FYqkI9r._0wfN4orDJ5wVHlNlVfITQ8NZ6EM(EnumRenderStage ,
Single )
в
_ahKHCLkUrtGs3H3UuMzDUmqqXDB._0wfN4orDJ5wVHlNlVfITQ8NZ6EM(EnumRenderStage ,
Single )
в _ahKHCLkUrtGs3H3UuMzDUmqqXDB._PKHjJEFCRCayAa1PJiCgoFdq0BAA(Single )
в _dUVWkWoBzqfq0KarNubKsO4pwLJ._PKHjJEFCRCayAa1PJiCgoFdq0BAA(Single )
в _zvL0LAdwCEhHXqW6hBlHn28VNoV._1avYKdA0wJOA4HAvdrnp9neEyJo(Single )
в _zvL0LAdwCEhHXqW6hBlHn28VNoV._IYaCeaicCgeLpRc26VEy79NrqA2b(Single )
в _B7H7nOajzVR29X1m2PTKbDfgYoq._U9eP5ZzkrZlktAYajF5LSjGAvmA(Object ,
FrameEventArgs )
в System.EventHandler`1.Invoke(Object sender, TEventArgs e)
в OpenTK.GameWindow.RaiseRenderFrame(Double elapsed, Double& timestamp)
в OpenTK.GameWindow.DispatchRenderFrame()
в OpenTK.GameWindow.Run(Double updates_per_second, Double
frames_per_second)
в
_2G50b8IBBzglLvhk9KmePvBQJz4A._yZA5T6nTHB2CuNXWShkZH7I7UHp(_I1adoYFf4hNSpGjYe69kh6rErhH ,
String[] )
в _eFYeOXMx6SGc6gQ6NMWiHK2bPab._yZA5T6nTHB2CuNXWShkZH7I7UHp(ThreadStart )
------------------------------- ```

**Comment by Leden** [comment]
Also, after breaking the aforementioned vessel, crashed like this. Additionally, now crashing when logging in into that chunk.
```
Running on 64 bit Windows with 8 GB RAM
Version: v1.10.6 (Stable)19/09/15 5:34:10 PM: Critical error occurred
System.NullReferenceException: Ссылка на объект не указывает на экземпляр
объекта.
в Vintagestory.GameContent.MealMeshCache.GenMealMesh(CookingRecipe
forRecipe, ItemStack[] contentStacks, Vec3f foodTranslate)
в Vintagestory.GameContent.MealMeshCache.GenMealInContainerMesh(Block
containerBlock, CookingRecipe forRecipe, ItemStack[] contentStacks, Vec3f
foodTranslate)
в
Vintagestory.GameContent.MealMeshCache.GetOrCreateMealInContainerMeshRef(Block
containerBlock, CookingRecipe forRecipe, ItemStack[] contentStacks, Vec3f
foodTranslate)
в Vintagestory.GameContent.BlockMeal.OnBeforeRender(ICoreClientAPI capi,
ItemStack itemstack, EnumItemRenderTarget target, ItemRenderInfo&
renderinfo)
в
Vintagestory.Client.NoObf.InventoryItemRenderer.GetItemStackRenderInfo(_ahKHCLkUrtGs3H3UuMzDUmqqXDB
game, ItemSlot inSlot, EnumItemRenderTarget target)
в Vintagestory.GameContent.EntityItemRenderer.DoRender3DOpaque(Single
dt, Boolean isShadowPass)
в
Vintagestory.Client.NoObf.SystemRenderEntities.OnRenderFrameShadows(Single
dt)
в
_ee95dOUYyEAX8VgbT4J2FYqkI9r._0wfN4orDJ5wVHlNlVfITQ8NZ6EM(EnumRenderStage ,
Single )
в
_ahKHCLkUrtGs3H3UuMzDUmqqXDB._0wfN4orDJ5wVHlNlVfITQ8NZ6EM(EnumRenderStage ,
Single )
в _ahKHCLkUrtGs3H3UuMzDUmqqXDB._s8slRCQtDSRQI6yQ7WHN5S73NHH(Single )
в _ahKHCLkUrtGs3H3UuMzDUmqqXDB._xNlL8C9SVADdWoGFryPgDIDaBqc(Single )
в _dUVWkWoBzqfq0KarNubKsO4pwLJ._ssC94ZRubksGrGLXMFmSzddrCKM(Single )
в _zvL0LAdwCEhHXqW6hBlHn28VNoV._1avYKdA0wJOA4HAvdrnp9neEyJo(Single )
в _zvL0LAdwCEhHXqW6hBlHn28VNoV._IYaCeaicCgeLpRc26VEy79NrqA2b(Single )
в _B7H7nOajzVR29X1m2PTKbDfgYoq._U9eP5ZzkrZlktAYajF5LSjGAvmA(Object ,
FrameEventArgs )
в System.EventHandler`1.Invoke(Object sender, TEventArgs e)
в OpenTK.GameWindow.RaiseRenderFrame(Double elapsed, Double& timestamp)
в OpenTK.GameWindow.DispatchRenderFrame()
в OpenTK.GameWindow.Run(Double updates_per_second, Double
frames_per_second)
в
_2G50b8IBBzglLvhk9KmePvBQJz4A._yZA5T6nTHB2CuNXWShkZH7I7UHp(_I1adoYFf4hNSpGjYe69kh6rErhH ,
String[] )
в _eFYeOXMx6SGc6gQ6NMWiHK2bPab._yZA5T6nTHB2CuNXWShkZH7I7UHp(ThreadStart )
------------------------------- 
```

**Comment by tyronx** [comment]
thanks should be fixed in v1.10.7
